// juniper.afi_tree_entry is generated by proto_generator as a protobuf
// representation of a YANG schema.
//
// Input schema modules:
//  - afi-cap-action.yang
//  - afi-cap-entry-action.yang
//  - afi-cap-entry-match.yang
//  - afi-cap-entry.yang
//  - afi-cap-match.yang
//  - afi-cap.yang
//  - afi-common.yang
//  - afi-cos-classifier.yang
//  - afi-counter.yang
//  - afi-decap.yang
//  - afi-discard.yang
//  - afi-encap-entry.yang
//  - afi-encap.yang
//  - afi-filter.yang
//  - afi-fwd-sandbox.yang
//  - afi-indexed-list.yang
//  - afi-indirect.yang
//  - afi-list.yang
//  - afi-match.yang
//  - afi-oam.yang
//  - afi-policer.yang
//  - afi-port.yang
//  - afi-receive.yang
//  - afi-replicate.yang
//  - afi-sample.yang
//  - afi-sandbox.yang
//  - afi-selector.yang
//  - afi-switch.yang
//  - afi-table.yang
//  - afi-tree-encap-entry.yang
//  - afi-tree-encap.yang
//  - afi-tree-entry.yang
//  - afi-tree.yang
//  - afi-types.yang
//  - afi.yang
syntax = "proto3";

package juniper.afi_tree_entry;

import "ywrapper/ywrapper.proto";
import "yext/yext.proto";

message AfiTreeEntry {
  ywrapper.StringValue name = 298281935;
  ywrapper.StringValue parent_name = 84475654;
  ywrapper.StringValue prefix_bytes = 312714340;
  ywrapper.UintValue prefix_length = 35426751;
  ywrapper.StringValue target_afi_object = 227270412;
  ywrapper.UintValue vrf_id = 245987046;
}
